storage/sqlite.go

```diff
@@ -370,6 +371,17 @@ func (s *Sqlite) updateUser(name string, r *bool, st *api.Status) error {
 		return err
 	}
 
+	tx := db.Updates(updates)
+
+	if err := tx.Error; err != nil {
+		return err
+	}
+
+	totalAffectedRows := tx.RowsAffected
+	if totalAffectedRows == 1 { //chon vase 1 usere
+		updates["LastUserStatusUpdateTime"] = time.Now().Format(time.StampMicro)
```

Comment:
It doesn't affect DB. You update the `updates` map but you don't update DB with changes of `updates`.